src/devices/tuya.ts

```diff
@@ -3709,6 +3725,25 @@ const definitions: Definition[] = [
             ],
         },
     },
+    {
+        fingerprint: [
```

Comment:
As all code is the same except the picture, you can just use a `tuya.whitelabel` (look in this file for examples)

Reply:
Hi, battery is different:
```battery.low()```  vs ```batteryState()``` : binary vs. enum